infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/provision/KubernetesCheApiInternalEnvVarProvider.java

```diff
@@ -28,7 +28,8 @@
   private final String cheServerEndpoint;
 
```

Comment:
Javadoc for this class should be updated ("URL of Che API" is unclear if we're supporting both public endpoint and internal service).